src/scss/colorways.scss

```
@@ -1145,6 +1145,43 @@
   color: $color-gmk-abs-N9;
 }
 
+.gmk-vaporwave-key {
+  $color-vaporwave-lighter-pink: scale-color(
+    #ff3eb5,
+    $lightness: 60%,
+    $saturation: -20%
+  );
+  $color-vaporwave-lighter-lilac: scale-color(#9595d2, $lightness: 5%);
+  background-color: $color-vaporwave-lighter-lilac;
+  color: $color-vaporwave-lighter-pink;
+  font-weight: bold;
+  input,
+  .key-contents {
+    background: lighten($color-vaporwave-lighter-lilac, 5%);
+    color: $color-vaporwave-lighter-pink;
+    border-color: mix(
+      $color-vaporwave-lighter-lilac,
+      $color-vaporwave-lighter-pink
+    );
+  }
+}
+
+.gmk-vaporwave-mod {
+  background-color: #6244bb;
+  color: #00bce3;
+  input,
+  .key-contents {
+    background: lighten(#6244bb, 10%);
```

Color shift this the other way to increase the contrast.

```suggestion
    background: darken(#6244bb, 10%);
```